pkg/gofr/datasource/pubsub/mqtt/mqtt_test.go

```diff
@@ -561,3 +562,28 @@ func getMockMQTT(t *testing.T, conf *Config) (*gomock.Controller, *MQTT, *MockCl
 
 	return ctrl, mq, mockClient, mockMetrics, mockToken
 }
+
+func TestMQTT_Close(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	ctx := context.TODO()
```

Comment:
It's a test. Todo is for helping people to add context in code

Here you could use


```suggestion
	ctx := context.Background()
```